odin/tests/services/test_context.py

```
@@ -58,3 +60,12 @@ def test_update_index_context_cache(self, mock_subprocess):
         with patch.object(cache, "set") as mock_set:
             update_index_context_cache()
             mock_set.assert_called_once()
+
+        def test_traffic_included_in_index_context(self):
```

Ruff (0.14.14)

[warning] Unused function argument: `self`

(ARG001)